src/patternfly/components/Popover/popover-title.hbs

```diff
@@ -1,9 +1,6 @@
-<div class="pf-c-popover__title{{#if popover-title--IsIcon}} pf-m-icon{{/if}}{{#if popover-title--modifier}} {{popover-title--modifier}}{{/if}}"
+<div class="pf-c-popover__title{{#if popover-title--modifier}} {{popover-title--modifier}}{{/if}}"
   {{#if popover-title--attribute}}
```

Comment:
Since you're having to repeat `id`, I prefer adding `id` to dedicated `.hbs` params. 

```suggestion
  {{#if popover-title--id}}
    id="{{popover-title--id}}"
  {{/if}}
  {{#if popover-title--attribute}}
```